### PR TITLE
[Machine Learning] Multi-Output Classification with Machine Learning

### LINKED ISSUE #5535: [Machine Learning] Multi-Output Classification with Machine Learning
#### Proposed title of article
[Machine Learning] Multi-Output Classification with Machine Learning

#### Proposed article introduction
Multi-Output classification is a type of machine learning that predicts two different outputs at the same time. In Multi-Class machine learning, the model usually predicts only a single output but in multi-output classification, we have more than one output. Therefore, the model will be able to give more than one prediction at the same time.

For Example,  a multi-output classification model predicts the “type of fruit” and “color” of fruit at the same time.  The “type of fruit” cab be, “orange”, “mango” and “pineapple”. The “color” can be, “red”, “green”, “yellow”, and “orange". The multi-output classification can be used to solve this problem and give two prediction results.

In this tutorial, we will build a machine learning model that will be able to classify the `genre` of a movie or a series such as `drama`, `comedy`, `action`, or `thriller`.  The model will also be able to predict the `rating` of a movie or series such as `PG`, `TV14` or `TV18`. The output after prediction will be both the `genre` and `rating` of movies or series. We will use Scikit-Learn `MultiOutputClassifier` to build this model.

#### Key takeaways 
1. Multi-Class vs Multi-Label vs Multi-Output classification.
2. Techniques used to handle multi-output classification.
3.  Dataset preprocessing(Netflix dataset).
4. Building the multi-output prediction model.
5. Model predictions. (Predicts the `genre` and `rating` of movies or series)

#### Article quality
This tutorial is unique because we will compare the types of classification, Multi-Class vs Multi-Label vs Multi-Output classification. This will give us a deeper understanding of the problems before we start working on Multi-Output classification.  We will build a custom model using  Scikit-Learn `MultiOutputClassifier` algorithm. This is a practical approach and it will explain all the steps to build the model.
#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by willyngashu**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5535 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by WanjaMIKE** [comment]
Plagiarism check done.

File: content/articles/multi-output-classification-with-machine-learning/index.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+Before we start building our model, we also need to clean our dataset. Dataset cleaning involves correctly formatting our dataset.
+
+### Text cleaning
+
+For text cleaning, we will convert all our text data into lower case and also remove stop words. We will use the NeatText Python package to perform this process. We will install Neattext using the following code:
+
+```python
+! pip install neattext
+```
```
Comment (Bayler) [inline content/articles/multi-output-classification-with-machine-learning/index.md:111]: This is a bash command

File: content/articles/multi-output-classification-with-machine-learning/index.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+- Understand [natural langauge processing](/engineering-education/nlp-based-detection-model-using-neattext-and-scikit-learn/)
+- Know how to train models using [Scikit-learn](https://scikit-learn.org/stable/)
+- Use [Google Colab notebook](https://research.google.com/) to run the Python code in this tutorial.
+
+### Netflix dataset
+We will use the Netflix dataset to build our model. The image below will show how our dataset is structured.
+
+![Dataset image](/engineering-education/multi-output-classification-with-machine-learning/dataset-image.png)
+
+From the image above our dataset has four columns: `title`, `description`, `type`, and `rating`. The `title` column will be the input column while type` and `rating` will the be output column. We now need to load this dataset on our machine.
```
Comment (Bayler) [inline content/articles/multi-output-classification-with-machine-learning/index.md:37]:
```suggestion
From the image above, our dataset has four columns: `title`, `description`, `type`, and `rating`. The `title` column will be the input column while type` and `rating` will be the output column. We now need to load this dataset on our machine.
```

File: content/articles/multi-output-classification-with-machine-learning/index.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
+Multi-output classification is a type of machine learning that predicts multiple outputs at the same time. In multi-output classification, the model will give two or more outputs after making any prediction. In other types of classifications, the model usually predicts only a single output.
+
+An example of a multi-output classification model is a model that predicts the “type of fruit” and “color” of fruit at the same time. The “type of fruit” cab be, “orange”, “mango” and “pineapple”. The “color” can be, “red”, “green”, “yellow”, and “orange". The multi-output classification is used to solve this problem and gives two prediction results.
```
Comment (Bayler) [inline content/articles/multi-output-classification-with-machine-learning/index.md:3]:
```suggestion
An example of a multi-output classification model is a model that predicts the “type” and “color” of fruit at the same time. The “type of fruit” can be, “orange”, “mango” and “pineapple”. The “color” can be, “red”, “green”, “yellow”, and “orange". The multi-output classification is used to solve this problem and gives two prediction results.
```

**Pull request closed without merging**